pageserver/src/tenant/storage_layer/filename.rs

```diff
@@ -210,9 +210,15 @@ pub enum LayerFileName {
 
 impl LayerFileName {
     pub fn file_name(&self) -> String {
+        self.to_string()
+    }
+}
```

Comment:
Trying to remove or make this private reveals some uses, which I think could be using the Display... Or is there some reason for them not to migrate?

Reply:
Mhh yeah my initial search didn't catch all of the places where `file_name` was used in a debug setting. I've done another search now and removed the `file_name` calls where I could.